patches/tModLoader/Terraria/Main.cs.patch

```diff
@@ -1523,6 +1524,15 @@
 +						ItemLoader.GetWeaponCrit(item, player[myPlayer], ref crit);
 +						PlayerHooks.GetWeaponCrit(player[myPlayer], item, ref crit);
 +						toolTipLine[numLines] = crit + Lang.tip[5].Value;
++						toolTipNames[numLines] = "CritChance";
++						numLines++;
++					}
++					*/
++					if (!item.summon) {
++						int crit = player[myPlayer].GetWeaponCrit(item);
++						ItemLoader.GetWeaponCrit(item, player[myPlayer], ref crit);
++						PlayerHooks.GetWeaponCrit(player[myPlayer], item, ref crit);
++						toolTipLine[numLines] = crit + Lang.tip[5].Value;
```

Comment:
You need to revert this change too. I'm also happy to merge when you do. This is an example of why it's best to split features into separate PRs because one might be approved and the other need to wait for a redesign after discussion

Reply:
fixed up, should be good for real this time now